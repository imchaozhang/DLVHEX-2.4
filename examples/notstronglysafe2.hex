p(X) :- &testNonmon[p](X).
