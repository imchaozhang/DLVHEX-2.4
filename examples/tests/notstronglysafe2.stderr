1 grep -q "not strongly safe"
